package.json

```diff
@@ -52,6 +52,8 @@
     "eslint-plugin-jsx-a11y": "^5.1.1",
     "eslint-plugin-react": "^7.3.0",
     "jest-cli": "^19.0.2",
+    "jest-snapshot": "^24.8.0",
+    "react-addons-test-utils": "15.3.1",
```

Comment:
I'm a little surprised to see these? `enzyme` should cover test utils and I didn't think we were using `jest-snapshot` in this module?

Reply:
I was getting an error when trying to build the library. Maybe I misconfiguration something when I installed the repo ... or maybe the jest tests were always broken and we didn't notice because they weren't part of the travis build.

Reply:
I think the problem is that I was developing this on a SS45 install, which as `silverstripe/admin`which is running under react 16. The Travis build installs SS43.

Reply:
Yeah ... switch to using SS 4.3 and it worked fine.